### DISCUSSION TITLE
Add new extension

### DISCUSSION THREAD

**Discussion by xXCats** [aniyomiorg/aniyomi#784] [Ideas]
Hi among the anime extensions there is no 2 of the following sources where I usually watch anime:
Aniplay and AnimeUnity.tv you could add them.

https://www.animeunity.tv

-Thank you

**Comment by xXCats** [comment]
someone who can create the extension??

**Reply by 12ohit** [reply]
Request here: https://github.com/jmir1/aniyomi-extensions/issues/new?assignees=&labels=Source+request&template=request_source.yml
